arangod/WasmServer/WasmCommon.h

```diff
@@ -32,33 +32,29 @@
 
 namespace arangodb::wasm {
 
-struct WasmContainer {
-  uint8_t* code;
-  size_t length;
+struct Code {
+  std::vector<uint8_t> bytes;
```

Comment:
Did you check that the data in vector is guaranteed to be a continous chunk of memory that you can hand to `parse_module`?

Reply:
> The elements are stored contiguously, which means that elements can be accessed not only through iterators, but also using offsets to regular pointers to elements. This means that a pointer to an element of a vector may be passed to any function that expects a pointer to an element of an array. 
(https://en.cppreference.com/w/cpp/container/vector)
Additionally, same site says that it meets the requirements for ContiguousContainer which is a container that stores objects in contiguous memory locations.
(As far as I can see, only the specialization std::vector<bool> does no need to be contiuous for improved space efficiency (https://en.cppreference.com/w/cpp/container/vector_bool))

Reply:
:+1: